tests/integration/test_models.py

```diff
@@ -0,0 +1,357 @@
+                    author_id=1,
+                    author_name="Doe, John",
+                    publication=Publication(
+                        abstract="Many curated authors",
+                        authors=[
+                            "Doe, John",
+                            "Doe, J",
+                            "Doe, John",
+                            "Doe, John",
+                            "Doe, John",
+                            "Doe, John",
+                            "Jamie",
+                            "Jamie",
+                        ],
+                        collaborations=[],
+                        keywords=["keyword"],
+                        publication_id=1,
+                        title="Title",
+                        topics=["category"],
+                    ),
+                    signature_block="JOhn",
+                    signature_uuid="94fc2b0a-dc17-42c2-bae3-ca0024079e52",
+                    is_curated_author_id=True
+                ),
+                Signature(
+                    author_affiliation="Rutgers U., Piscataway",
+                    author_id=2,
+                    author_name="Doe, John",
+                    publication=Publication(
+                        abstract="Many curated authors",
+                        authors=[
+                            "Doe, John",
+                            "Doe, J",
+                            "Doe, John",
+                            "Doe, John",
+                            "Doe, John",
+                            "Doe, John",
+                            "Jamie",
+                            "Jamie",
+                        ],
+                        collaborations=[],
+                        keywords=["keyword"],
+                        publication_id=1,
+                        title="Title",
+                        topics=["category"],
+                    ),
+                    signature_block="JOhn",
+                    signature_uuid="94fc2b0a-dc17-42c2-bae3-ca0024079e54",
+                    is_curated_author_id=True
+                ),
+            ],
+        ],
+        dtype=object,
+    )
+    expected_y = array([0, 0, 1, 1])
+    assert (distance_estimator.X == expected_X).all()
+    assert (distance_estimator.y == expected_y).all()
+
+
+@patch("inspire_disambiguation.core.ml.models.DistanceEstimator")
+@patch("inspire_disambiguation.core.es.readers.LiteratureSearch.scan")
+def test_clusterer_load_data(
+    scan_mock,
+    distance_estimator_mock,
+    es_record_with_curated_author,
+    es_record_with_non_curated_author,
+):
+    scan_mock.side_effect = [
+        [es_record_with_curated_author, es_record_with_non_curated_author]
+    ]
+    signatures = get_signatures()
+    input_clusters = get_input_clusters(signatures)
+    clusterer = Clusterer(distance_estimator_mock)
+    clusterer.load_data(signatures, input_clusters)
+    expected_X = array(
```

Comment:
why ``X`` uppercase and ``y`` lowercase?

Reply:
It's `sklearn` convention.